### PR TITLE
inline contentType field fixes

### PR DESCRIPTION
**Pull request by andreasplesch**
there were some mixups

### CODE REVIEW

File: src/nodes/Networking/Inline.js [hunk 1]
```diff
@@ -171,7 +171,7 @@ x3dom.registerNodeType(
             {
                 for ( var type in this.ContentType )
                 {
-                    if( contentType === this.ContentType[ type ] )
+                    if( type === contentType )
```
Comment (tsturm) [inline src/nodes/Networking/Inline.js]: if( contentType === this.ContentType[ type ] ) is the correct one here.
Input is "model/x3d+xml", "model/gltf+json", "model/gltf-binary" or "model/x3d+json"
The range of the documentation was wrong at that point

File: src/nodes/Networking/Inline.js [hunk 2]
```diff
@@ -185,7 +185,7 @@ x3dom.registerNodeType(
                 //Return contentType if defined & valid
                 if (this._vf.contentType != "" && this.isValidContentType( this._vf.contentType ) )
                 {
-                    return contentType;
+                    return this.ContentType[this._vf.contentType];
```
Comment (tsturm) [inline src/nodes/Networking/Inline.js]: It should be: return this._vf.contentType;

I already commit the change to master

**Comment by tsturm** [comment]
Already fixed with f65cecf11771fbca46217a218f3461c70eddd699

**Comment by andreasplesch** [comment]
ok

**Pull request closed without merging**